src/Illuminate/Encryption/EncryptionPayload.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+namespace Illuminate\Encryption;
+
+
+class EncryptionPayload implements \JsonSerializable
+{
+
+    private $iv;
```

Comment:
Would it be worth to least add a doc block for these variables to state their type?